packages/create-cloudflare/e2e-tests/frameworks/framework-test-config.ts

```diff
@@ -394,6 +395,38 @@ export default function getFrameworkTestConfig(pm: string) {
 				"@/*",
 			],
 		},
+		"next:workers": {
+			argv: ["--platform", "workers"],
+			testCommitMessage: true,
+			flags: [
+				"--ts",
+				"--tailwind",
+				"--eslint",
+				"--app",
+				"--import-alias",
+				"@/*",
+				"--src-dir",
+			],
+			verifyBuildCfTypes: {
+				outputFile: "cloudflare-env.d.ts",
+				envInterfaceName: "CloudflareEnv",
+			},
+			verifyPreview: {
+				route: "/test",
+				expectedText: "Create Next App",
+			},
+			verifyDeploy: {
+				route: "/",
+				expectedText: "Create Next App",
+			},
+			// see https://github.com/cloudflare/next-on-pages/blob/main/packages/next-on-pages/docs/supported.md#operating-systems
```

Comment:
Is this comment (and the Windows exclusion) actually valid for OpenNext?
Not for this PR but perhaps a quick follow-up? cc @vicb